packages/langium/src/test/langium-test.ts

```diff
@@ -306,3 +327,8 @@ export function expectWarning<T extends AstNode = AstNode, N extends AstNode = A
         ...content,
     });
 }
+
+function CleanUpTestDocuments(services: LangiumServices) {
```

Comment:
Suggestion: I believe we should export this, so that downstream users can easily clean up their tests, even when they're not using our testing functions.

```suggestion
export function clearDocuments(services: LangiumServices) {
```